### ISSUE TITLE
wrong openjdk version

### ISSUE THREAD

**Issue by pwkooij** [beagle-dev/beagle-lib#103]
While trying to configure I receive the following error:

`configure: error: no acceptable Java compiler found in $PATH`

I think this has to do with the fact that openjdk-6 is required, however, the oldest version I seem to be able to get is openjdk-7 with the standard installation being openjdk-8. Trying to deploy on Ubuntu 16.04.

Any idea what to do?

**Comment by msuchard** [comment]
Hi!  If you could provide me with a `Dockerfile` that builds a container that reproduces this issue, I can try to debug it for you.

**Comment by pwkooij** [comment]
I seem to be having some firewall problems while working with docker, trying to solve that. In the meantime I noticed this in the build file [https://github.com/beagle-dev/beagle-lib/blob/master/build.xml](url)

`<target name="compile" depends="init">`
`	<!-- Compile the java code from ${src} into ${build} -->`
`	<javac source="1.6" target="1.6" srcdir="${src}" destdir="${build}" classpath="">`
`		<include name="beagle/**"/>`
`	</javac>`
`</target>`

could the version directed to in there be a problem?

PS will get the Dockerfile as soon as I solve the firewall issues

**Comment by pwkooij** [comment]
I seem to be having some firewall problems while working with docker, trying to solve that. In the meantime I noticed this in the build file [https://github.com/beagle-dev/beagle-lib/blob/master/build.xml](url)

`<target name="compile" depends="init">`
`	<!-- Compile the java code from ${src} into ${build} -->`
`	<javac source="1.6" target="1.6" srcdir="${src}" destdir="${build}" classpath="">`
`		<include name="beagle/**"/>`
`	</javac>`
`</target>`

could the version directed to in there be a problem?

PS will get the Dockerfile as soon as I solve the firewall issues

**Comment by pwkooij** [comment]
I finally managed to get docker to work, and when I tried to reproduce the problem, there was no problem with java there. Trying again on my normal terminal I noticed it couldn't find `javac`, which made me suspect something was wrong with my java installation. I purged all java applications, and reinstalled the openjdk-8 from the ubuntu repository, and now it works! 

So never mind the other comments. Cheers
